apps/minifront/src/state/staking/index.ts

```diff
@@ -70,6 +70,7 @@ export interface StakingSlice {
    * `delegationsByAccount`. Should be called each time `account` is changed.
    */
   loadDelegationsForCurrentAccount: () => Promise<void>;
+  loadDelegationsForCurrentAccountAbortController?: AbortController;
```

Comment:
Good lord that's a long var name

Reply:
Haha, yeah. I was thinking of calling it just `abortController`, but I wanted it to be clear what it's being used for.